plugin-base/src/main/java/io/everytrade/server/parser/exchange/GbApiTransactionBean.java

```diff
@@ -161,7 +162,8 @@ public TransactionCluster toTransactionCluster() {
                 quoteCurrency,
                 TransactionType.valueOf(action),
                 quantity,
-                volume.divide(quantity, 10, RoundingMode.HALF_UP)
+                volume.divide(quantity, 10, RoundingMode.HALF_UP),
+                getRemoteUid()
```

Comment:
Why we are parsing uid to get note?

Reply:
Because you wanted to create the note only from 'remote' part of uid. From the GB connector comes uid 'local'-'remote' together.